public/static/docs/api-reference/index.md

```diff
@@ -0,0 +1,18 @@
+# Python API
+
+When you [install](/doc/install) DVC in an environment, the `dvc` package
+becomes available for usage in Python source code. While most of its code is the
```

Comment:
Actually guys @Suor @shcheklein I installed DVC with `brew` as recommended for MacOS and I do not have access to the `dvc` package from `python` or `python3`. Does it have to be installed with `pip` in order for the API to be available in an environment?

Reply:
yes, it should be part of the requirements/setup file and should be installed via some env manager like pip, conda, etc

Reply:
OK, added note to index, please check.